core/store/orm/schema.go

```diff
@@ -20,7 +20,9 @@ type ConfigSchema struct {
 	ClientNodeURL             string         `env:"CLIENT_NODE_URL" default:"http://localhost:6688"`
 	DatabaseTimeout           time.Duration  `env:"DATABASE_TIMEOUT" default:"500ms"`
 	DatabaseURL               string         `env:"DATABASE_URL"`
+	DefaultHTTPAttempts       uint           `env:"DEFAULT_HTTP_ATTEMPTS" default:"5"`
```

Comment:
Maybed we could brainstorm a better name, we'll be stuck with this name for a while

Reply:
Yeah now that i think about it "Default' really makes no sense. How about just `MaxHTTPAttempts` ?

Reply:
On second thought, maybe `DefaultMaxHTTPAttempts` because we could override this value in the future.